### PR TITLE
Auto

### PR DESCRIPTION
**Pull request by yapei**
Ready for review @yanpzhan

### CODE REVIEW

File: lib/rules/web/admin_console/4.6/monitoring.xyaml [hunk 1]
```diff
@@ -33,4 +33,96 @@ browse_to_receiver_config:
...
+      xpath: //tbody//td[text()='namespace']
+  - selector:
+      xpath: //tbody//td[text()='pod_usage_cpu_core_seconds']
+check_usage_report_table:
+  action: goto_one_report_page
+  action:
+    if_param:
+      report_name: namespace-cpu-usage
+    ref: check_namespace_cpu_usage_usage_report_table
+check_namespace_cpu_usage_usage_report_table:
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.6/monitoring.xyaml:101]: two "usage" in this rule name?

Comment (yapei) [reply lib/rules/web/admin_console/4.6/monitoring.xyaml:101]: @yanpzhan  Ah yes, the report name is `namespace-cpu-usage` and the table name is `Usage Report`. Do you think I need to change to other name?

**Comment by yapei** [comment]
@pruan-rht Would you help review and approve it if it looks good to you?

**Comment by yapei** [comment]
@liangxia  Could you help review and merge? Thank you

**Pull request merged**